zerver/tests/test_messages.py

```diff
@@ -671,6 +671,20 @@ def test_message_mentions(self) -> None:
         message = most_recent_message(user_profile)
         assert(UserMessage.objects.get(user_profile=user_profile, message=message).flags.mentioned.is_set)
 
+    def test_is_private_flag(self) -> None:
+        user_profile = self.example_user('iago')
+        self.subscribe(user_profile, "Denmark")
+
+        self.send_stream_message(self.example_email("hamlet"), "Denmark",
+                                 content="test")
+        message = most_recent_message(user_profile)
+        self.assertFalse(UserMessage.objects.get(user_profile=user_profile, message=message).flags.is_private.is_set)
+
+        self.send_personal_message(self.example_email("hamlet"), user_profile.email,
+                                   content="test")
+        message = most_recent_message(user_profile)
+        assert(UserMessage.objects.get(user_profile=user_profile, message=message).flags.is_private.is_set)
```

Comment:
`self.assertTrue` is the right way to write this.

Reply:
(In general, one doesn't use the `assert` keyword in unit tests; it produces less nice output than the `self.assert*` methods.

Reply:
Cool, I'll push the change with the other changes and will take care of this is future :)